README.md

```diff
@@ -2,17 +2,24 @@
 
 Source code repo for the pelux.io website
 
-## How to build
+## How to build the web site locally
 
-This website uses [jekyll](https://jekyllrb.com/) to build static HTML. These are the steps to build the web site locally:
-*NB: You'll want to have jekyll and bundler installed on your system first.*
+This website uses [jekyll](https://jekyllrb.com/) to build static
```

Comment:
Is there a specific reason why you added new lines here? I understand why one does it for code but for text it is not really necessary because all text editors can handle breaking into new lines very well and if you change the text you don't need to manually reformat the whole paragraph because the text editor will do that for you.